### DISCUSSION TITLE
How to avoid logging Non-Fatal events on Firebase Crashlytics

### DISCUSSION THREAD

**Discussion by akiladevinda** [invertase/react-native-firebase#5536] [Q&A]
Hi All, 

I have added firebase crashlytics to my project and on my Crashlytics dashboard there are Non-Fatal expectations also logged in. I only want to log App crashes only and how to prevent non-fatal exceptions logging to crashlytics.

My firebase.json file

```
{
    "crashlytics_debug_enabled": false,
    "crashlytics_disable_auto_disabler": true,
    "crashlytics_auto_collection_enabled": true,
    "crashlytics_is_error_generation_on_js_crash_enabled": true,
    "crashlytics_javascript_exception_handler_chaining_enabled": true
}
```

**Comment by mikehardy** [comment]
This document explains the available toggle for you: https://rnfirebase.io/crashlytics/usage#crashlytics-javascript-stacktrace-issue-generation

Also maybe of interest https://invertase.io/blog/react-native-firebase-crashlytics-configuration

**Reply by akiladevinda** [reply]
Thank you
